README.md

```diff
@@ -143,6 +143,10 @@ vscode ➜ /workspaces/fasttrackml (main) $ make run
 ```
 </details>
 
+### Performance benchmarking
+Here is a performance benchmark comparison between FasttrackML and MLflow
+![Performance Report](benchmarks/performanceReport.png)
+
```

Comment:
Sorry for the changing direction -- I think we want to take this out of the main README until we're feeling really good about the benchmarks

Reply:
Okay thats fine